server/go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	go.opentelemetry.io/otel/sdk v1.20.0
 	go.uber.org/multierr v1.6.0
 	go.uber.org/zap v1.17.0
-	golang.org/x/crypto v0.14.0
+	golang.org/x/crypto v0.17.0
```

Comment:
Please bump to 0.18.0 so that it's >= the version being used in 3.4, and also ensure all modules use the same version of each dependency (golang.org/x/crypto in this case)

Reply:
SG

Reply:
Managed to rework the https://github.com/etcd-io/etcd/pull/17347 managed to upgrade crypto to just 0.17.0 on release-3.4

Reply:
> Managed to rework the #17347 managed to upgrade crypto to just 0.17.0 on release-3.4

Did you run into any issue when bumping to v0.18.0? A higher version is recommended.

Reply:
Upgrading crypto to v0.18.0 required bumping grpc which I wanted to avoid. My goal was to do a minimal dependency bump that addresses the CVE. I would prefer to rather bump crypto to 0.17.0 (minimal required to address CVE) than bump crypto to 0.18.0 and grpc to 1.60.

Reply:
OK